ckanext/datajson/package2pod.py

```diff
@@ -3,23 +3,20 @@
 except ImportError:
     from sqlalchemy.util import OrderedDict
 
-import string
+import collections
+import json
 import os
+import string
 import sys
-import json
-import ckan.plugins.toolkit as tk
-import collections
-import ckan.plugins.toolkit as toolkit
-
 from logging import getLogger
 
-from helpers import *
-from ckan import logic
-from ckan import model
 import ckan.plugins as p
-import ckan
-import urllib
```

Comment:
I don't see these added back in anywhere; are they just unnecessary or were they removed purposefully?

Reply:
they weren't being used